ingestion/src/metadata/ingestion/ometa/mixins/es_mixin.py

```diff
@@ -84,12 +90,10 @@ def es_search_from_fqn(
         )
 
         try:
-            entity_list = self._search_es_entity(
+            query_string, response = self._search_es_entity(
```

Comment:
@ayush-shah any specific reason for returning the `query_string` from `_search_es_entity` method? I don't see it being used anywhere?

Reply:
this is for the LRU cache mechanism, it needs a tuple response having, one of the params and other being the response.

in this case, if it hits, the cache will automatically return the response

Reply:
let me make that change, LRU is running with that as well.